src/fides/api/alembic/migrations/versions/ddec29f24945_update_cookie_max_age_to_bigint.py

```diff
@@ -0,0 +1,26 @@
+"""update cookie max age to bigint
+
+Revision ID: ddec29f24945
+Revises: c5a218831820
+Create Date: 2023-10-24 13:05:40.442124
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import Integer
+from sqlalchemy.dialects.postgresql import BIGINT
+
+# revision identifiers, used by Alembic.
+revision = "ddec29f24945"
+down_revision = "c5a218831820"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column("ctl_systems", "cookie_max_age_seconds", type_=BIGINT)
```

Comment:
@adamsachs here is the migration that we talked about

Reply:
I was able to upgrade and downgrade successfully with data in that column. I also manually changed the type back and forth within pgadmin and it worked there too